storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java

```diff
@@ -99,7 +99,46 @@ public void nonEmptyListWithNullElementToUnmodifiableNonNullListOfStrings() {
 
     //endregion
 
-    //region Tests for Queries.unmodifiableNonNullList()
+    //region Tests for Queries.unmodifiableNonNullList() from array of objects
+
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void nullArrayToUnmodifiableNonNullList() {
+        Object[] array = null;
+        assertThat(InternalQueries.unmodifiableNonNullList(array)).isEqualTo(emptyList());
```

Comment:
I replaced it with `isSameAs` to check not just size, but also that it is unmodifiable